WordPress/src/test/java/org/wordpress/android/viewmodel/pages/CreatePageListItemLabelsUseCaseTest.kt

```diff
@@ -0,0 +1,365 @@
+package org.wordpress.android.viewmodel.pages
+
+import com.nhaarman.mockitokotlin2.anyOrNull
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.PostModel
+import org.wordpress.android.fluxc.model.page.PageStatus.DRAFT
+import org.wordpress.android.fluxc.model.post.PostStatus
+import org.wordpress.android.fluxc.model.post.PostStatus.PENDING
+import org.wordpress.android.fluxc.model.post.PostStatus.PRIVATE
+import org.wordpress.android.fluxc.model.post.PostStatus.PUBLISHED
+import org.wordpress.android.fluxc.model.post.PostStatus.SCHEDULED
+import org.wordpress.android.fluxc.store.MediaStore.MediaError
+import org.wordpress.android.fluxc.store.MediaStore.MediaErrorType.AUTHORIZATION_REQUIRED
+import org.wordpress.android.fluxc.store.PostStore.PostError
+import org.wordpress.android.fluxc.store.PostStore.PostErrorType
+import org.wordpress.android.fluxc.store.PostStore.PostErrorType.GENERIC_ERROR
+import org.wordpress.android.fluxc.store.UploadStore.UploadError
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.viewmodel.pages.CreatePageUploadUiStateUseCase.PostUploadUiState
+
+@RunWith(MockitoJUnitRunner::class)
+class CreatePageListItemLabelsUseCaseTest {
+    @Mock private lateinit var pageConflictResolver: PageConflictResolver
+    @Mock private lateinit var labelColorUseCase: ResolvePageListItemsColorUseCase
+    private lateinit var useCase: CreatePageListItemLabelsUseCase
+
+    @Before
+    fun setUp() {
+        useCase = CreatePageListItemLabelsUseCase(
+                pageConflictResolver,
+                labelColorUseCase
+        )
+    }
+
+    @Test
+    fun `private label shown for private pages`() {
+        val (labels, _) = useCase.createLabels(
+                PostModel().apply { setStatus(PRIVATE.toString()) },
+                mock()
+        )
+        assertThat(labels).contains(UiStringRes(R.string.page_status_page_private))
+    }
+
+    @Test
+    fun `pending review label shown for pages pending review`() {
+        val (labels, _) = useCase.createLabels(
+                PostModel().apply { setStatus(PENDING.toString()) },
+                mock()
+        )
+        assertThat(labels).contains(UiStringRes(R.string.page_status_pending_review))
+    }
+
+    //
```

Comment:
Were you planning to put a comment here? Let me know 😄

Reply:
Ohh, I missed that :P - Fixed in e8ce3a0cdb03c96fb66f2af70156f70148fefe3b